docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -13,103 +13,103 @@ weight: 150
 To send OTel data to Grafana Cloud, refer to [Send data using OpenTelemetry Protocol (OTLP)](https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/).
 {{% /admonition %}}
 
-When using the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can write metrics into Mimir via two options: `prometheusremotewrite` and `otlphttp`.
+When using the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can write metrics into Mimir using two options: `otlphttp` (preferred) and `prometheusremotewrite`.
 
-We recommend using each protocol's respective exporter and native Mimir endpoint:
+Use each protocol's respective exporter and native Mimir endpoint. For example:
 
-- Prometheus data: `prometheusremotewrite`
 - OTel data: `otlphttp`
+- Prometheus data: `prometheusremotewrite`
 
-## Prometheus Remote Write
-
-For the Remote Write, use the [`prometheusremotewrite`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusremotewriteexporter) exporter in the Collector:
+## OTLP
 
-In the `exporters` section add:
+Mimir supports native OTLP over HTTP. When possible, use this protocol to send OTLP data. To configure the collector to use the OTLP interface, use the [`otlphttp`](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlphttpexporter) exporter:
 
 ```yaml
 exporters:
-  prometheusremotewrite:
-    endpoint: http://<mimir-endpoint>/api/v1/push
+  otlphttp:
+    endpoint: http://<mimir-endpoint>/otlp
```

Comment:
This differs between grafana cloud and OSS mimir.

If its OSS mimir this is correct.

If its Grafana Cloud we want the users to write to the otlp-endpoint/otlp check this doc for context https://grafana.com/docs/grafana-cloud/send-data/otlp/send-data-otlp/

Reply:
Thanks for the additional details. Since these are the Mimir docs, is it fair to assume that folks are using Mimir and not Grafana Cloud here?